crates/autopilot/src/database/onchain_order_events.rs

```diff
@@ -600,7 +600,12 @@ fn convert_onchain_order_placement(
         order_data.fee_amount
     };
     let is_outside_market_price = if let Ok(ref quote) = quote {
-        if is_order_outside_market_price(&order_data.sell_amount, &order_data.buy_amount, quote) {
+        if is_order_outside_market_price(
```

Comment:
nit: something like `if !order.within_market(quote)` may be nicer to read

Reply:
refactored.